.doc_gen/cross-content/cross_RDSDataTracker_cpp_block.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "file://zonbook/docbookx.dtd"[
+    <!ENTITY % phrases-shared SYSTEM "file://AWSShared/common/phrases-shared.ent">
+    %phrases-shared;
+]>
+<block>
+    <para>
+        Shows how to create a web application that tracks and reports on work items stored in an &RDS; database. 
```

Comment:
I would change this from "&RDS; database" to "&AURlong; Serverless database" because that's how the service team refers to the feature.